Adding special character caveat to PG DBM doc (#23095)
DataDog/documentation: content/en/database_monitoring/setup_postgres/selfhosted.md

{{% /tab %}}
{{< /tabs >}}

> **Note**: Wrap your password in single quotes in case a special character is present.

Reviewer: shall we add this note to all deployment types?

> **Note**: Wrap your password in single quotes in case a special character is present.

Reviewer 2: ```suggestion
**Note**: Wrap your password in single quotes if special character is present.
```
